### ISSUE TITLE
"Similar Artists" missing -- LastFM API changes?

### ISSUE THREAD

**Issue by jailbrokebloke** [airsonic-advanced/airsonic-advanced#468]
Has something changed with the LastFM API that might be preventing the normal list of "Similar Artists" from showing? This was working weeks ago, but now is showing blank. In the console log (StompClient msg) I can see before the "artistBio" info that the "similarArtists" array is empty...

Running version 11.0.0-SNAPSHOT.20201007202626, Apache Tomcat/9.0.38, java 11.0.9.1.

Thanks for any help.

**Comment by The-Exterminator** [comment]
The version 11.0.0-SNAPSHOT.20210210200712 this missing to

**Comment by jailbrokebloke** [comment]
> The version 11.0.0-SNAPSHOT.20210210200712 this missing to
> 
> Maybe, you could have the star made so that an artist photo comes on. ??

Are you saying the "Similar Artists" is missing for you too, like I described?

Strange no one has even confirmed the issue, and it's been 7 days.

**Comment by The-Exterminator** [comment]
Yes, the artist text is fine on the new version, But the artist picture have missions a Long time now.

**Comment by lbesnard** [comment]
>  but now is showing blank

To me this seems like a frontend issue. I can't see the "similar artists", "top songs" and "start radio"; features I'm using a lot. 
However these features are accessible via dsub on Android
